integreat_cms/cms/utils/pdf_utils.py

```diff
@@ -195,3 +196,25 @@ def link_callback(uri, rel):
             finders.searched_locations,
         )
     return result
+
+
+def sanitize_for_pdf(html):
+    """
+    Helper function for sanitizing HTML for use with xhtml2pdf,
+    which lacks some features commonly found in browsers.
+
+    :param html: rendered HTML page
+    :type html: str
+
+    :return: sanitized version of html param
+    :rtype: str
+    """
+    # keep the first occurrence, since it's intentionally
+    # set in page_pdf.html
+    html_parts = html.partition("font-family: ")
+    html = (
+        html_parts[0]
+        + html_parts[1]
+        + re.sub(r"font-[a-zA-Z]+:.*?;", "", html_parts[2])
+    )
+    return html
```

Comment:
In my small test this worked as expected, but I'm not sure whether it's robust enough against invalid HTML (which we unfortunately might have in our content), e.g. if the semicolon is missing after the font definition, some more content than desired might be removed etc...
```suggestion
    content = fromstring(html)
    for element in content.iter():
        # Remove all inline font style definitions
        style = element.attrib.pop("style", None)
        if style:
            element.attrib["style"] = re.sub(r"font-[a-zA-Z]+:.*?;", "", style)
    return tostring(content, with_tail=False).decode("utf-8")
```
(This requires `from lxml.html import fromstring, tostring`)

Reply:
Yeah, you are absolutely right. Your suggestion is a lot safer, and does away with partitioning out the wanted styling.